.github/workflows/main.yml

```
@@ -87,3 +87,115 @@ jobs:
+                      username: sa
+                      password: Admin12345
+                      database: master
+                      # no double quotes — the health command is itself wrapped in them below
+                      health: /opt/mssql-tools18/bin/sqlcmd -S localhost -U sa -P Admin12345 -C -Q 'SELECT 1'
+                    - driver: mongodb
+                      image: mongo:8
+                      port: 27017
+                      username: ''
+                      password: ''
+                      database: test
+                      health: mongosh --quiet --eval 'db.adminCommand({ ping: 1 })'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)